dash/include/dash/halo/Halo.h

```diff
@@ -898,205 +901,203 @@ class HaloBlock {
    */
   const PatternT& pattern() const { return _pattern; }
 
-  const GlobMemT& globmem() const { return _globmem; }
+  const GlobMem_t& globmem() const { return _globmem; }
 
-  const region_t* halo_region( const region_index_t index ) const {
+  const Region_t* halo_region(const region_index_t index) const {
     return _halo_reg_mapping[index];
   }
 
-  const regions_t& halo_regions() const { return _halo_regions; }
+  const RegionVector_t& halo_regions() const { return _halo_regions; }
 
-  const region_t* boundary_region( const region_index_t index ) const {
+  const Region_t* boundary_region(const region_index_t index) const {
     return _boundary_reg_mapping[index];
   }
 
-  const regions_t& boundary_regions() const { return _boundary_regions; }
+  const RegionVector_t& boundary_regions() const { return _boundary_regions; }
 
   // const halo_regs_t& halo_regions() const { return _halo_regions; }
 
-  const ViewSpecT& view() const { return _view; }
+  const ViewSpec_t& view() const { return _view; }
 
-  const ViewSpecT& view_safe() const { return _view_safe; }
+  const ViewSpec_t& view_safe() const { return _view_safe; }
```

Comment:
Just out of curiosity: you mentioned that you are looking for a different name for `safe`, any luck so far?

Reply:
view_fool_prove? :)

Reply:
but serious: how view_reliable sound?

Reply:
Right now, I can't think of the exact meaning of the safe view anymore (missing documentation, y'a know ;)). I vaguely remember that this was the view on the halo cells (?) Why not call it that?

Reply:
is inner + boundary to guarantee neighbours -> thats why it's called view_guaranteed now